custom/enikshay/management/commands/data_dumps_person_case.py

```diff
@@ -105,6 +119,39 @@ def get_all_episode_cases_from_person(domain, person_case_id):
     ]
 
 
+def get_all_episode_cases_from_occurrence(domain, occurrence_case_id):
+    return CaseAccessors(domain).get_reverse_indexed_cases(
+        [occurrence_case_id], case_types=[CASE_TYPE_EPISODE])
+
+
+def get_last_occurrence(person_case):
+    occurrence_cases = get_all_occurrence_cases_from_person(person_case.domain, person_case.case_id)
+    open_occurrence_cases = [
+        occurrence_case for occurrence_case in occurrence_cases
+        if not occurrence_case.closed
+    ]
```

Comment:
Feel free to punt on this, but FYI, `get_reverse_indexed_cases` also accepts an `is_closed` argument, if you wanna avoid doing this in memory.

Reply:
Need all occurrence_cases in there are no open occurrence cases so i think fine to not do that here.